README.md

```diff
@@ -30,38 +20,25 @@ different containerized cloud infrastructure deployments.
 
 ## Introduction
 
-Containers are superior to conventional deployment mechanism in that they are:
-
-1. Isolated:
-  - Applications have their own libraries, no conflicts will arise from
-  different libraries in other applications.
-
+Containers are superior to conventional deployment mechanisms in that they are:
```

Comment:
Can we rephrase this to something like: There are many advantages to building your deployment pipeline around containers... It's a nit, but saying that containers are always superior seems like a hard claim to back up and is a little far reaching, IMO

Reply:
100% agreed. Probably an overly-light touch for me to leave that in.